docs/releases/release_process.md

```diff
@@ -137,14 +137,19 @@ varying levels.
     -   Major functionality bugs in newly-introduced features.
 
     -   Regressions from older versions of Wagtail.
+ 
+-   At the discretion of maintainers, we may also apply patches in additional scenarios:
+ 
+    -   Accessibility bugs, blocking specific user groups.
+
+    -   User interface bugs, adding unreasonable friction.
 
     The rule of thumb is that fixes will be backported to the last feature
     release for bugs that would have prevented a release in the first place
-    (release blockers).
+    (release blockers). Additionally, fixes for the above issues may also be backported to releases under "active support" at the discretion of maintainers.
 
 -   Security fixes and data loss bugs will be applied to the current `main`, the
-    last feature release branch, and any other supported long-term
-    support release branches.
+    last feature release branch, and any other releases under "security support".
```

Comment:
Suggestion: Move this bullet above the new bullet ("At the discretion of…") so it's clear that we will always do this and there is no discretionary space intended for these cases.